packages/framework/presence/src/presenceManager.ts

```diff
@@ -30,16 +30,26 @@ import type {
 import { createEmitter } from "@fluid-experimental/presence/internal/events";
 
 /**
+ * Requirements for components connecting inside (under) the presence manager.
+ *
  * @internal
  */
-export interface IPresenceManager
-	extends IPresence,
-		Pick<Required<IContainerExtension<[]>>, "processSignal"> {}
+export type PresenceInternal = Pick<IPresence, "getAttendee">;
+
+/**
+ * Requirements for presence manager to be connected as an extension.
```

Comment:
Short description around "connected as an extension" would be nice to have here.